scripts/rhel_validation/check_lis_modules.sh

```diff
@@ -31,15 +31,15 @@ main() {
         printf "\nLoaded LIS modules after modprobe:\n"
     fi
     for module in ${UNSET_MOD[@]};do
-        dmesg --clear
+        dmesg -c 1> /dev/null 2>&1
```

Comment:
can you please let me know why changing this?
with "-c 1" the output is printed, while with '--clear' it will not display, but here you're displaying and sending that to dev/null, basically what --clear does.

Reply:
dmesg --clear does not work for RH6. I have used other approach to clear the dmesg and redirect both stdout and stderr to /dev/null. This will work for RH6 and RH7.